.bp-config/php/php.ini.d/memory_limit.ini

```diff
@@ -0,0 +1,3 @@
+; Maximum amount of memory a script may consume (128MB)
+; http://php.net/memory-limit
+memory_limit = 512M
```

Comment:
Oof, this is pretty low. It might be OK, but if we're seeing issues with migrations etc, I'd look here first.

Reply:
The migrations with issues are failing because of a timeout of 180 seconds for build operations run via the cloudfoundry CLI. The migrations that complete within three minutes like agencies and goals finish without issues. The ones that take longer like plan_files and report_files fail due to timeout. The timeout maximum is unfortunately not impacted by this setting and I think we need to get with cloud.gov support to change. Will reach out to them.

64M is the minimum [php memory requirement](https://www.drupal.org/docs/getting-started/system-requirements/overview) for Drupal 10 so 512M seemed generous but definitely down to bump this up if we're seeing other issues.